library/src/main/kotlin/build/buf/connect/impl/ProtocolClient.kt

```diff
@@ -117,6 +120,28 @@ class ProtocolClient(
         }
     }
 
+    override fun <Input : Any, Output : Any> unarySync(
+        request: Input,
+        headers: Headers,
+        methodSpec: MethodSpec<Input, Output>
+    ): Call<Output> {
+        val countDownLatch = CountDownLatch(1)
+        val reference = AtomicReference<ResponseMessage<Output>>()
+        // Create a new Call object.
+        val call = Call<Output>()
+        // Set the unary synchronous executable.
+        call.setExecute { callback: (ResponseMessage<Output>) -> Unit ->
+            val cancellationFn = unary(request, headers, methodSpec) { responseMessage ->
```

Comment:
Not sure if we use `cancellation` or `cancelation` in this repo, but just want to make sure we're consistent everywhere